app/src/androidTest/java/com/example/henzoshimada/feeltrip/FeelTripTests.java

```diff
@@ -45,7 +45,7 @@ public void testAddMoodTask(){
         ElasticSearchController.AddMoodTask addMoodTask = new ElasticSearchController.AddMoodTask();
         Mood mood = new Mood("user1");
         try {
-            mood.setDescription("description1");
+            mood.setDescription("description1", " -Feeling sleepy");
```

Comment:
So setDescription() takes two arguments now. A String for the actual user description, and a String for the "-Feeling _____" that we want to append to the end. This does not have to function this way. Instead of passing "-Feeling ____" and storing it into Elasticsearch, we could simply append it to the end every time we display the actual message.